### ISSUE TITLE
Add averages to results exports

### ISSUE THREAD

**Issue by janno42** [e-valuation/EvaP#2001]
In all Excel results exports, an additional column should be inserted after the questions (in column B), showing the average of all answers given for this question in all evaluations.

The new column should have the header "Average for this question". The lines "degrees", "course type", and the summarizing lines on the bottom should be empty.

**Comment by fidoriel** [comment]
Agreed with @janno42 to only use published evaluations from all semesters.

**Comment by janno42** [comment]
Update: We only want to use Evaluations from non-archived semesters (`results_are_archived == False`) for the average.
